packages/next-loader/src/defineLive.tsx

```diff
@@ -179,7 +179,7 @@ export function defineLive(config: DefineSanityLiveOptions): {
     perspective?: Omit<ClientPerspective, 'raw'>
     tag?: string
   }) {
-    const stega = _stega ?? (await draftMode()).isEnabled
+    const stega = _stega ?? ((await draftMode()).isEnabled && client.config().stega)
```

Comment:
I'm not sure this works.

We want to allow this pattern:
```
const client = createClient({
  // ...
  stega: {
    studioUrl: '/studio' <- stega is ready, but not enabled
  }
})

export const {sanityFetch, SanityLive} = defineLive({client})
```

We don't want stega to be enabled in production, but if userland has to do this:
```
const client = createClient({
  // ...
  stega: {
    enabled: true, <- applies to both production and draft mode
    studioUrl: '/studio'
  }
})

export const {sanityFetch, SanityLive} = defineLive({client})
```